e2e-tests/cypress/e2e/Reporting-index.cy.ts

```diff
@@ -3,9 +3,14 @@ import { Loader } from '../support/page-objects/Loader';
 import { RorMenu } from '../support/page-objects/RorMenu';
 import { Discover } from '../support/page-objects/Discover';
 import { Settings } from '../support/page-objects/Settings';
+import { KibanaNavigation } from '../support/page-objects/KibanaNavigation';
+import * as semver from 'semver';
+import { getKibanaVersion } from '../support/helpers';
+import { kbnApiAdvancedClient } from '../support/helpers/KbnApiAdvancedClient';
+
+describe('Reporting index', () => {
```

Comment:
maybe we can give it a more descriptive name?

Reply:
I adjusted the description to `Index Pattern Configuration for Reporting index`